traffic_ops/app/db/admin.go

```diff
@@ -93,16 +104,20 @@ const (
 	// goose commands that don't match the commands for this tool
 	GooseUp = "up"
```

Comment:
This variable isn't used anywhere in the code.

Reply:
`GooseUp` removed in 5d4d8d4f27